### ISSUE TITLE
Explore: Group table frames with subframes

### ISSUE THREAD

**Issue by ifrost** [grafana/grafana#71530]
> The whole splitting logic could be encapsulated in a single function. Maybe not in this PR, but do you think it would make sense to make it part of the decorator logic?

_Originally posted by @ifrost in https://github.com/grafana/grafana/pull/70935#discussion_r1261097528_

**Comment by ifrost** [comment]
No longer relevant after #71953
